src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -603,21 +622,18 @@ export default class CanvasController {
         cc.lineJoin = 'round';
         cc.font = BASE_CANVAS_FONT;
 
-        _forEach(this._navigationLibrary.realFixes, (fix, i) => {
+        const fixes = this._navigationLibrary.realFixes;
```

Comment:
this assignment is unnecessary.  we can simply use `this._navigationLibrary.realFixes` within the for declaration and when assigning `fix`.

If we _did_ need it, we should actually use destructuring here: 
```javascript
const { realFixes } = this._navigationLibrary;
```